src/main/scala/scorex/util/serialization/Serializer.scala

```diff
@@ -0,0 +1,6 @@
+package scorex.util.serialization
+
+trait Serializer[TFamily, T <: TFamily, R <: Reader, W <: Writer] {
```

Comment:
Why do we need both `TFamily` and `T` here?

Reply:
used at https://github.com/Scorexfoundation/sigmastate-interpreter/blob/master/src/main/scala/sigmastate/serialization/ValueSerializer.scala